tests/test_cmd_test.py

```diff
@@ -75,7 +75,9 @@ def test_tool_test_timeout(self):
                 assert (
                     float(tool_test_json["tests"][0]["data"]["time_seconds"]) <= 10
                 ), "Test needed more than 10 sec but should time out after 1"
-                assert "Timed out after" in tool_test_json["tests"][0]["data"]["problem_log"], "Time out did not happen"
```

Comment:
This scares me - these logs were being cleaned up and merged by code removed? This will cause a degradation in the reports? Again... a question for me...

Reply:
FWIW we dump that in https://github.com/mvdbeek/galaxy/blob/3aed09379875dc48610fbe86b897d03c6b74b143/lib/galaxy/tool_util/verify/interactor.py#L1271. So workflow tests or tests against external engines have used this for years at this point.